modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -746,4 +757,22 @@ def setBuildStatus(repoName, commit, message, state) {
   ]);
 }
 
+def runPublishingE2ETests() {
+  build(
+    job: "publishing-e2e-tests/${params.PUBLISHING_E2E_TESTS_BRANCH}",
+    parameters: [
+      [$class: "StringParameterValue",
+        name: "PUBLISHING_API_COMMITISH",
```

Comment:
I think this needs to be just the application that is being tested.

E.g. CONTENT_STORE_COMMITISH in this instance. Which is something e2e tests would need to support